src/w3d/renderer/render2dsentence.cpp

```diff
@@ -862,7 +870,7 @@ void Render2DSentenceClass::Build_Sentence_Centered(const unichar_t *text, int *
 
                 if (cur_ch == U_CHAR(' ')) {
                     m_cursor.X += char_spacing;
-                } else if (cur_ch == U_CHAR('\n') || cur_ch == U_CHAR('\0')) {
+                } else if (cur_ch == U_CHAR('\0') || cur_ch == U_CHAR('\n')) {
```

Comment:
Why has this been swapped

Reply:
In the decompile it is in this order and therefore easier to compare against.